### PR TITLE
chore: mantine-fy loader styles

### PR DESCRIPTION
**Pull request by joaoviana**
Closes: N/A

### Description:

Update `Loader` styles (like the `PageSpinner`)

### Reviewer actions

- [ ] I have manually tested the changes in the preview environment
- [ ] I have reviewed the code
- [ ] I understand that "request changes" will block this PR from merging

### CODE REVIEW

File: packages/frontend/src/components/PageSpinner/index.tsx [hunk 1]
```diff
@@ -1,10 +1,10 @@
-import { Spinner } from '@blueprintjs/core';
-import React, { FC } from 'react';
+import { Loader } from '@mantine/core';
+import { FC } from 'react';
 import { LogoContainer, SpinnerWrapper } from './PageSpinner.styles';
 
 const PageSpinner: FC = () => (
     <SpinnerWrapper data-testid="page-spinner">
-        <Spinner size={100} />
+        <Loader color="gray.5" size={100} />
```
Comment (joaoviana) [inline packages/frontend/src/components/PageSpinner/index.tsx:7]: Let me know what you think @IrakliJani (cc @PriPatel )

See here example:

https://github.com/lightdash/lightdash/assets/7611706/55b3d6a6-334e-48cc-a67c-3a1d491927c7

Comment (IrakliJani) [reply packages/frontend/src/components/PageSpinner/index.tsx:7]: @joaoviana can we make it thinner. like.. way thinner?

Comment (joaoviana) [reply packages/frontend/src/components/PageSpinner/index.tsx:7]: IDK how! 🤣
checking!

Comment (PriPatel) [reply packages/frontend/src/components/PageSpinner/index.tsx:7]: Yeah it looks a bit thic

Comment (joaoviana) [reply packages/frontend/src/components/PageSpinner/index.tsx]: https://github.com/lightdash/lightdash/assets/7611706/8ed6e0a6-e99e-46c5-a6c5-87aa5dca5533

not thicc anymore!

File: packages/frontend/src/features/export/hooks/useExportToGoogleSheet.tsx [hunk 2]
```diff
@@ -25,12 +25,7 @@ const useExportToGoogleSheetStart = ({
                 showToast({
                     title: 'Exporting Google Sheets',
                     subtitle: 'This may take a few minutes...',
-                    icon: (
-                        <Spinner
-                            className="bp4-icon bp4-icon-error"
-                            size={16}
-                        />
-                    ),
+                    icon: <Loader color="dark" size="xs" />,
```
Comment (IrakliJani) [inline packages/frontend/src/features/export/hooks/useExportToGoogleSheet.tsx:28]: this is conflicting with my alerts change but its fine. gonna be a simple merge

**Review by IrakliJani — APPROVED** [review]
good stuff. one comment

File: packages/frontend/src/features/export/hooks/useExportToGoogleSheet.tsx (hunk 2 above)
Comment (IrakliJani) [reply packages/frontend/src/features/export/hooks/useExportToGoogleSheet.tsx:28]: more like a heads up :D

**Pull request merged**